make.bash

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+function help() {
+    echo "---- Project: Ptt-backend ----"
+	  echo " Usage: ./make.bash COMMAND"
+    echo
+    echo " Management Commands:"
+    echo "  build              Build project"
```

Comment:
I am thinking if we should have `clean` `install`

Reply:
```clean``` is  definitely a good idea. I will update it.
```install```  I can't image when to use this command, IMO Ptt-backend is not command line tool and the recommended way to run local service is  docker compose.  please correct me if anything woring.